share/spice/whois/whois.js

```diff
@@ -113,16 +115,25 @@
         return api_result.dataError && api_result.dataError === 'MISSING_WHOIS_DATA';
     }
 
-    //Converts timestamp into local time using getDateFromString
+    //Converts timestamp into local time using moment.js
     function prettifyTimestamp(timestamp) {
         if(!timestamp) { return; }
-        var dateObj = DDG.getDateFromString(timestamp),
-            monthArr = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"],
-            day = dateObj.getDate(),
-            month = monthArr[dateObj.getMonth()+1],
-            year = dateObj.getFullYear();
-        if(day<10) {day='0'+day};
-        return month + " " + day + ", " + year;
+
+        //When using the expiresDateNormalized (example: 2020-09-14 00:00:00 UTC); need to use custom date formatting
```

Comment:
@moollaza Sure, was awaiting this question :wink: 

Looking at the following [conditional block](https://github.com/MrChrisW/zeroclickinfo-spice/blob/whois-updates/share/spice/whois/whois.js#L76-L81) you can see we failback to using the `DateNormalized` values if we're unable to find `updatedDate` or `expiresDate` values.
 
moment.js cannot automatically parse the normalized dates so we need to provide a custom format `YYYY-MM-DD HH:mm:ss Z` [line 125](https://github.com/MrChrisW/zeroclickinfo-spice/blob/whois-updates/share/spice/whois/whois.js#L125)